### DISCUSSION TITLE
Recover backup

### DISCUSSION THREAD

**Discussion by joaochris** [sws2apps/organized-app#290] [Q&A] [answered]
I set up someone else's account and already gave them access. But it does not appear to recover backup. Is it the same or do I need to do some more steps?

(In this case, I already filled in all the data for parts and so on, in one account. And I want someone else to be able to recover the backup to continue from that data.)

**Comment by rhahao** [comment, marked as answer]
Hello @joaochris, thanks for reaching out! When you give access to that person, have you assigned as well the View Meeting Schedule role? If a person is already the Life and Ministry Meeting Overseer or Assistant, the View Meeting Schedule is not needed anymore. For now, these roles when mixed will cancel each other out.

**Reply by joaochris** [reply]
Got it, but is it possible to recover a backup created in the main account (in the account created with the congregation?) in another account with the assigned privileges?

**Comment by rhahao** [comment]
Yes, that's exactly how it works. The backup is saved for a congregation. So anyone with the correct role should be able to restore it.

**Reply by joaochris** [reply]
Now it's all buggy lol 😂
Only this appears on both accounts.
I lost functions. 

![Screenshot_3](https://github.com/sws2apps/cpe-sws/assets/76891864/466d0f12-2e1b-4a15-b40d-1654cec44fdc)

**Reply by rhahao** [reply]
😊I see your account was assigned the View Meeting Schedule role. As I explained above, this cancels out the features you receive as Life and Ministry Meeting Overseer. If you no longer see the Manage role page, I can update your role back to normal.

**Reply by joaochris** [reply]
I understand. I ended up selecting the wrong one. If you can restore I appreciate it!

**Reply by rhahao** [reply]
@joaochris: the role was updated. You can reload CPE now. Hope it will work as before after that.

**Reply by joaochris** [reply]
Now it worked for two accounts, and I managed to understand. Thanks :)
